Makefile.envs

```diff
@@ -129,8 +129,6 @@ export MAIN_MODULE_LDFLAGS= $(LDFLAGS_BASE) \
 	-s TOTAL_MEMORY=20971520 \
 	-s ALLOW_MEMORY_GROWTH=1 \
 	-s EXPORT_ALL=1 \
-	-s POLYFILL \
-	-s MIN_SAFARI_VERSION=140000 \
```

Comment:
Also, we still should specify the min Safari version for emscripten, shouldn't we?

Reply:
Well it is now going to default to `150000` which is the max version that Emscripten ever checks against. But I think we are in effect requiring Safari 16.4 because although the Safari 15.0 changelog reports adding support for externref, the Safari implementation is broken until version 16.4. Of course there is changelog entry to reporting the bugfix...